src/FrontDoor/FrontDoor/Cmdlets/NewAzureRmFrontDoorCustomRuleObject.cs

```diff
@@ -43,7 +42,8 @@ public class NewAzureRmFrontDoorCustomRuleObject : AzureFrontDoorCmdletBase
         /// Type of the rule. Possible values include: 'MatchRule', 'RateLimitRule'
         /// </summary>
         [Parameter(Mandatory = true, HelpMessage = "Type of the rule. Possible values include: 'MatchRule', 'RateLimitRule'")]
-        public PSCustomRuleType RuleType { get; set; }
+        [PSArgumentCompleter("RateLimitRule", "MatchRule")]
```

Comment:
You will need to add suppressions for these in BreakingChangeIssues.csv:  for your module. See: https://dev.azure.com/azure-sdk/29ec6040-b234-4e31-b139-33dc4287b756/_apis/build/builds/20929/artifacts?artifactName=analyze-windows&fileId=0DBAB6745C618379FABC826D4D0D3AD6DC4A155A553FDB175F94796C7AB5884201&fileName=BreakingChangeIssues.csv&api-version=5.0-preview.3

Reply:
@markcowl - Added suppressions to breakingchangeissues.csv